crates/client/src/client/types/block.rs

```diff
@@ -36,7 +36,8 @@ pub struct Header {
     pub transactions_count: u64,
     pub message_receipt_count: u64,
     pub transactions_root: MerkleRoot,
-    pub message_receipt_root: MerkleRoot,
+    pub message_outbox_root: MerkleRoot,
+    pub event_inbox_root: MerkleRoot,
```

Comment:
it doesn't look like this actually gets set by anything?

Reply:
I'm not sure what you mean. The only constructor I see is the `From`. Is there another place you'd expect it to be set?

Reply:
Ah I added this comment before I saw the latest commit